stdpopsim/catalog/HomSap/__init__.py

```diff
@@ -202,6 +202,37 @@
     )
     _species.add_genetic_map(_gm)
 
+###########################################################
+#
+# Annotations
+#
+#
+#
+###########################################################
+
+_an = stdpopsim.Annotation(
+    species=_species,
+    id="Ensembl_GRCh38_gff3",
+    description="Ensembl GFF3 annotations on GRCh38",
+    long_description="""
+        These are the complete GFF3 annotations from
+        Ensembl. Please see
+        ftp://ftp.ensembl.org/pub/grch37/current/gff3/homo_sapiens/README
+        """,
+    url=(
+        "ftp://ftp.ensembl.org/pub/release-101/"
+        "gff3/homo_sapiens/Homo_sapiens.GRCh38.101.gff3.gz"),
+    zarr_url=(
+        "https://stdpopsim.s3-us-west-2.amazonaws.com/"
+        "annotations/HomSap.zip"),
```

Comment:
How about including the ensembl version in the zarr zip filename? Consider what happens if a user has an old version in their cache, and then the genome assembly is updated -> no new annotation file is downloaded.